pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2567,4 +2587,72 @@
       ``[secure port host]
     ==
   ==
+++  return-static-data
+    |=  [code=@ content-type=@t data=octs]
+    ::
+    :*  %start
+        :-  status-code=code
+        ^=  headers
+          :~  ['content-type' content-type]
+              ['content-length' (crip (format-ud-as-integer p.data))]
+          ==
+        data=[~ data]
+        complete=%.y
+    ==
+++  handle-scry
```

Comment:
We've already talked about deduplicating this with the `+request:per-server-event` logic, just leaving a note here so we don't forget!

Reply:
yeah i'm in the process of figuring out why i'm getting empty scry results when i scry into other vanes through `per-server-event`. still crossing my fingers i can get my HD back because i can't quite remember how i fixed that.